# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../freeipmi/hotfix.patch
# Copyright (C) 2013 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./libipmimonitoring/ipmi_monitoring.h.in.orig	2013-06-26 09:01:19.775134921 +0200
+++ ./libipmimonitoring/ipmi_monitoring.h.in	2013-06-26 09:01:37.135136232 +0200
@@ -160,7 +160,7 @@
     IPMI_MONITORING_SENSOR_TYPE_OEM_MIN                             = 0xC0,
     IPMI_MONITORING_SENSOR_TYPE_OEM_MAX                             = 0xFF,
     IPMI_MONITORING_SENSOR_TYPE_UNKNOWN                             = 0xFFFF,
-  } ipmi_monitoring_sensor_type_t;
+  };
 
 enum ipmi_monitoring_sel_event_direction
   {
